src/shell/lean.cpp

```diff
@@ -289,6 +298,32 @@ options set_config_option(options const & opts, char const * in) {
     }
 }
 
+void load_plugin(std::string path) {
```

Comment:
We should move this function to a different folder since we need it to implement the `import plugin <plugin-name>` command